apps/src/netsim/NetSimBitLogPanel.js

```diff
@@ -174,3 +186,57 @@ NetSimBitLogPanel.prototype.setChunkSize = function (newChunkSize) {
   this.chunkSize_ = newChunkSize;
   this.render();
 };
+
+/**
+ * Sets the vertical space that this log panel should consume (including margins)
+ * @param {number} heightPixels
+ */
+NetSimBitLogPanel.prototype.setHeight = function (heightPixels) {
+  this.openHeight_ = heightPixels;
+  this.sizeToOpenHeight_();
```

Comment:
The `NetSimBitLogPanel` caches off the height it's given because its "clear" button actually triggers a re-render of the whole widget and we have to preserve this property across swapping out the markup.  You'll notice below that the `NetSimLogPanel` doesn't have to do this, as its "clear" button only empties the contents of its inner scroll area.